Add IoT class for Input Datetime

jordibsk10 opened the pull request:
As requested in #14661 this is to add the IoT class to Input Datetime.

## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

jordibsk10 committed:
Add IoT class for Input Datetime

frenck commented on source/_integrations/input_datetime.markdown:
```diff
@@ -4,6 +4,7 @@ description: Instructions on how to integrate the Input Datetime integration int
 ha_category:
   - Automation
 ha_release: 0.55
+ha_iot_class: Calculated
```
This integration does not calculate a state, nor does it get one from a device.

```suggestion
ha_iot_class: ~
```

This will set the IoT class to none/null explicitly

jordibsk10 committed:
Update source/_integrations/input_datetime.markdown

It get its state based on automations mainly, but it may come from a device' state or from anything you can think of with an automation, right? 😅) that's why I stated "Calculated".
But I probably misunderstood the meaning behind the "Calculated" class, apologies.
Thanks

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

frenck reviewed (approved):
Thanks, @jordibsk10 👍 

Happy Hacktoberfest 🎉
